website/content/docs/commands/events.mdx

```diff
@@ -0,0 +1,105 @@
+---
+layout: docs
+page_title: events - Command
+description: |-
+  The "events" command interacts with the Vault events subsystem.
+---
+
+# events
+
+The `events` command is used to subscribe to Vault
+[events](/vault/docs/concepts/events) via `events subscribe`. It shows a
+real-time display of events generated by Vault on standard out.
+
+The desired event types (sometimes called "topics") must be specified, which
+may be a glob pattern (with `*` treated as a wildcard).
+
+The output will be JSON objects serialized using the default protobuf JSON
+serialization format, with one line per event received.
+
+Note that this command is designed to run indefinitely. It is similar to
+`tail -f` in the Unix world. This command will not exit on its own unless it
+encounters an unexpected error. As a user, you must terminate this process
+yourself to shut it down.
+
+## Examples
+
+Subscribe to all events:
+
+```shell-session
+$ vault events subscribe '*'
+```
+
+Subscribe to all KV events:
+
+```shell-session
+$ vault events subscribe 'kv*'
+```
+
+Subscribe to all `kv-v2/data-write` events:
+
+```shell-session
+$ vault events subscribe kv-v2/data-write
+```
+
+Subscribe to all non-`data-write` KV events for the secret `secret/data/foo` in
+the current namespace as well as namespace `ns1`:
+
+```shell-session
+$ vault events subscribe -namespaces=ns1 -filter='data_path == secret/data/foo and operation != "data-write"' 'kv*'
+```
+
+## Usage
+
+The following flags are available in addition to the [standard set of
+flags](/vault/docs/commands) included on all commands.
+
+### Options
+
+- `-timeout`: `(duration: "")` - close the WebSocket automatically after the
+  specified duration.
+
+- `-filter` `(string: "")` - Filter expression used to select events to be sent
+  through the WebSocket.
+
+  See [Boundary's docs](/boundary/docs/concepts/filtering) on how these
+  expressions are evaluated and what options are available.
+
+  The following values are available in the filter expression:
+  - `event_type`: the event type, e.g., `kv-v2/data-write`.
+	- `operation`: the operation name that caused the event, e.g., `write`.
+	- `source_plugin_mount`: the mount of the plugin that produced the event,
+    e.g., `secret/`
+	- `data_path`: the API path that can be used to access the data of the secret related to the event, e.g., `secret/data/foo`
+	- `namespace`: the path of the namespace that created the event, e.g., `ns1/`
+
+  By default, the filter will be empty and will match all events that the
+  request has access to for the given event types specified in the request.
+
+  A filter could be as simple as `data_path == secret/data/foo` to specify that
+  only events that refer to the `data_path` of `secret/data/foo` should be sent
+  via the WebSocket.
+
+  A filter can also be more complex, for example:
+  ```
```

Comment:
I think the code fence should be typed as `shell-session` here.

Reply:
It's not a shell-session though. It's a generic boolean expression in the hashicorp/go-bexpr language.